src/elements/flag/flag.js

```diff
@@ -0,0 +1,19 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('semantic.ui.elements.flag', [])
+        .directive('smFlag', smFlag);
+
+    function smFlag($compile) {
+        return {
+            restrict: 'E',
+            replace: true,
+            scope: { value: '='},
+            link: function (scope, element) {
+                element.html('').append($compile('<i class="flag ' + scope.value + '"></i>')(scope));
```

Comment:
Hey, so this is only going to run one when compiled. It has an isolate scope whose value will be updated, but the dom itself won't be updated when the flag changes. Because of this, and because of the simplicity of the component, what if we remove the new scope and instead just interpolate the attribute string in the context of the parent scope?